modules/navier_stokes/doc/content/source/problems/NavierStokesProblem.md

```diff
@@ -88,6 +88,66 @@ A final option available to users is the [!param](/Problem/NavierStokesProblem/u
 in which case a "standard" preconditioner can be used (e.g. `-pc_type lu`,
 `-pc_type hypre -pc_hypre_type boomeramg`, etc.) as opposed to LSC.
 
+## Which preconditioner to choose
+
+[!citep](zanetti2020scalable) compares the Elman and Olshanskii LSC
+preconditioners for "high" Reynolds numbers. The overarching result from that
+paper is that Olshanskii is better performing, although the largest kinematic
+viscosity explored in that article direclty comparing the two methods is
+$10^{-1}$. The Olshanskii preconditioner is known to be sub-optimal for high
+viscosities. In the limit of Stokes flow in which the advective term is dropped,
```

Comment:
What do you mean by "high viscosities"? Slow fluid?

Reply:
It doesn't necessarily have to be slow, but it tends to be for high viscosity (like maple syrup)

Reply:
in this test input, the peak velocity is the same for all cases